README.md

```diff
@@ -55,10 +55,15 @@ let
 in
 pkgs.stdenv.mkDerivation rec {
   # < ... >
+  src = ./.;
```

Comment:
this should be `src = ./src;`, because the idea is that you directly consume sources without any directory structure. in addition, because requiring root sources is just not what i think anyone should do without a filter